SingularityUI/package.json

```diff
@@ -62,7 +62,10 @@
     "singularityui-tailer": "0.3.13",
     "typeahead.js": "^0.11.1",
     "underscore": "~1.8.0",
-    "uuid": "^3.0.0"
+    "uuid": "^3.0.0",
+    "xterm": "^4.3.0",
+    "xterm-addon-attach": "^0.4.0",
```

Comment:
this dependency currently breaks `gulp build` due to a stray typescript file in the package causing a syntax error in uglifyjs